hail/python/test/hailtop/batch/test_batch.py

```diff
@@ -686,23 +688,37 @@ def test_verbose(self):
         res_status = res.status()
         assert res_status['state'] == 'success', str((res_status, res.debug_info()))
 
-    def test_cloudfuse(self):
+    def test_cloudfuse_fails_with_read_write_mount_option(self):
         assert self.bucket
         path = f'/{self.bucket}{self.cloud_output_path}'
 
         b = self.batch()
-        head = b.new_job()
-        head.command(f'mkdir -p {path}; echo head > {path}/cloudfuse_test_1')
-        head.cloudfuse(self.bucket, f'/{self.bucket}', read_only=False)
+        j = b.new_job()
+        j.command(f'mkdir -p {path}; echo head > {path}/cloudfuse_test_1')
+        j.cloudfuse(self.bucket, f'/{self.bucket}', read_only=False)
 
-        tail = b.new_job()
-        tail.command(f'cat {path}/cloudfuse_test_1')
-        tail.cloudfuse(self.bucket, f'/{self.bucket}', read_only=True)
-        tail.depends_on(head)
+        try:
+            b.run()
+        except aiohttp.ClientResponseError as e:
+            assert e.message == 'Only read-only cloudfuse requests are supported.', e.message
+        else:
+            assert False
```

Comment:
I recently learned about `with pytest.raises(...):`. This is fine, but thought I'd share!